nextgen/project-expander/tests/tasks_expander_test.rs

```diff
@@ -809,6 +809,53 @@ mod tasks_expander {
                     ]
                 );
             }
+
+            #[test]
+            fn skip_missing_targets() {
```

Comment:
can you also add a test where it doesnt skip if the target does exist, and optional = true